src/app/pages/proof/proof.page.html

```diff
@@ -58,20 +58,25 @@ <h2>{{ t('information') }}</h2>
     </ion-list-header>
     <ion-item>
       <ion-slides pager>
-        <ion-slide *ngFor="let providerWithInformationList of (providersWithInformationList$ | async)">
-          <ion-card class="slide-card">
+        <ion-slide *ngFor="let listinfo of (information$ | async)">
```

Comment:
This can be simplified. There is no need to use two variables, `information$` and `providersWithInformationList$`, to display the expected view. We can merge these two variables into one Observable: `providersWithImportantInformation$`.

For instance,

``` typescript
readonly providersWithImportantInformation$ = this.proof$.pipe(
  switchMap(proof => this.informationRepository.getByProof$(proof)),
  map(informationList => {
    const providers = new Set(informationList.map(information => information.provider)); // Use Set to eliminate duplicated providers.
    return [...providers].map(provider => ({
      provider,
      informationList: informationList.filter(
        information => information.provider === provider
          && information.importance === Importance.High
      )
    }));
  })
);
```